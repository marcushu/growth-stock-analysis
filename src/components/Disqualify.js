import React from 'react';
import Row from 'react-bootstrap/Row';
import Col from 'react-bootstrap/Col';
import DividedByForm from './DividedByForm';

const Disqualify = () => {
  return (
    <div className="shadow p-3" style={{ backgroundColor: "rgb(255, 255, 255)" }}>
      <Row>
        <Col className="text-center">
          <h3>Quick Pre-qualify</h3>
          <hr style={{ border: "3px solid rgb(86, 141, 212)" }} />
        </Col>
      </Row>
      <Row>
        <Col>
          <h5><span className="font-weight-bolder">Market cap</span></h5>
          <br />
          <h5><span className="font-weight-bolder">Trading volume</span></h5>
          <br />
          <h5><span className="font-weight-bolder">Trailing twelve month cash flow</span></h5>
          <br />
          <h5><span className="font-weight-bolder">Sales</span> </h5>
        </Col>
        <Col>
          <h5>&gt; 50 million</h5> <br />
          <h5> &gt; 50,000 shares</h5> <br />
          <h5>positive</h5> <br />
          <h5>greater than 40 mi over last four quarters</h5> <br />
        </Col>
      </Row>
      <Row>
        <Col>
          <h5><DividedByForm /></h5>
        </Col>
      </Row>
    </div>
  )

}

export default Disqualify