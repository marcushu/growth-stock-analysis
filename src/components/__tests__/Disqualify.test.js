import React from 'react';
import { render,  cleanup } from '@testing-library/react';
import Disqualify from '../Disqualify';

afterEach(cleanup);

test("Disqualify renders ok", () => {
  const { asFragment } = render(<Disqualify />);

  expect(asFragment()).toMatchSnapshot();
});
