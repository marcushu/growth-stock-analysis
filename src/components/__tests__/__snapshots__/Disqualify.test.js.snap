// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Disqualify renders ok 1`] = `
<DocumentFragment>
  <div
    class="shadow p-3"
    style="background-color: rgb(255, 255, 255);"
  >
    <div
      class="row"
    >
      <div
        class="text-center col"
      >
        <h3>
          Quick Pre-qualify
        </h3>
        <hr
          style="border: 3px solid rgb(86, 141, 212);"
        />
      </div>
    </div>
    <div
      class="row"
    >
      <div
        class="col"
      >
        <h5>
          <span
            class="font-weight-bolder"
          >
            Market cap
          </span>
        </h5>
        <br />
        <h5>
          <span
            class="font-weight-bolder"
          >
            Trading volume
          </span>
        </h5>
        <br />
        <h5>
          <span
            class="font-weight-bolder"
          >
            Trailing twelve month cash flow
          </span>
        </h5>
        <br />
        <h5>
          <span
            class="font-weight-bolder"
          >
            Sales
          </span>
           
        </h5>
      </div>
      <div
        class="col"
      >
        <h5>
          &gt; 50 million
        </h5>
         
        <br />
        <h5>
           &gt; 50,000 shares
        </h5>
         
        <br />
        <h5>
          positive
        </h5>
         
        <br />
        <h5>
          greater than 40 mi over last four quarters
        </h5>
         
        <br />
      </div>
    </div>
    <div
      class="row"
    >
      <div
        class="col"
      >
        <h5>
          <form
            class=""
          >
            <div
              class="form-row"
            >
              <label
                class="form-label col-form-label col"
              >
                <span
                  class="font-weight-bolder"
                >
                  price/sales
                </span>
                 3 - 9
              </label>
              <div
                class="pb-1 col-xl"
              >
                <input
                  class="form-control"
                  style="width: 140px; margin-right: 20px; border: 0px; background-color: rgb(227, 227, 227);"
                  value="0"
                />
              </div>
              <div
                class="col-xl"
              >
                <input
                  class="form-control"
                  style="width: 140px; margin-right: 20px; border: 0px; background-color: rgb(227, 227, 227);"
                  value="0"
                />
              </div>
              <label
                class="form-label col-form-label col"
              >
                <div
                  style="color: rgb(86, 141, 212);"
                >
                  0.0
                </div>
              </label>
            </div>
          </form>
        </h5>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
